compliance_checker/cf/cf_1_7.py

```diff
@@ -794,7 +794,8 @@ def _evaluate_towgs84(self, val):
             return (True, msg)
 
     def check_grid_mapping(self, ds):
-        super().check_grid_mapping.__doc__
+        # FIXME: Looks like this is not needed.
+        # super().check_grid_mapping.__doc__
```

Comment:
Docstring is taken from `super` here or?

I guess Python 3.5 introduced docstring inheritance for subclass methods?  Probably wrote this back when we were supporting Python 2.7.

Reply:
To be honest I'm not sure what to do here.

Reply:
This is the behaviour of the docs without this line:

```python
Signature: CF1_7Check.check_grid_mapping(self, ds)
Docstring:
5.6 When the coordinate variables for a horizontal grid are not
longitude and latitude, it is required that the true latitude and
longitude coordinates be supplied via the coordinates attribute. If in
addition it is desired to describe the mapping between the given
coordinate variables and the true latitude and longitude coordinates,
the attribute grid_mapping may be used to supply this description.

This attribute is attached to data variables so that variables with
different mappings may be present in a single file. The attribute takes
a string value which is the name of another variable in the file that
provides the description of the mapping via a collection of attached
attributes. This variable is called a grid mapping variable and is of
arbitrary type since it contains no data. Its purpose is to act as a
container for the attributes that define the mapping.

The one attribute that all grid mapping variables must have is
grid_mapping_name which takes a string value that contains the mapping's
name. The other attributes that define a specific mapping depend on the
value of grid_mapping_name. The valid values of grid_mapping_name along
with the attributes that provide specific map parameter values are
described in Appendix F, Grid Mappings.

When the coordinate variables for a horizontal grid are longitude and
latitude, a grid mapping variable with grid_mapping_name of
latitude_longitude may be used to specify the ellipsoid and prime
meridian.


In order to make use of a grid mapping to directly calculate latitude
and longitude values it is necessary to associate the coordinate
variables with the independent variables of the mapping. This is done by
assigning a standard_name to the coordinate variable. The appropriate
values of the standard_name depend on the grid mapping and are given in
Appendix F, Grid Mappings.

:param netCDF4.Dataset ds: An open netCDF dataset
:rtype: list
:return: List of results
Source:   
    def check_grid_mapping(self, ds):
        # FIXME: Looks like this is not needed.
        # super().check_grid_mapping.__doc__
        prev_return = super().check_grid_mapping(ds)
        grid_mapping_variables = cfutil.get_grid_mapping_variables(ds)
        for var_name in sorted(grid_mapping_variables):
:
```

Is that OK?